assets/queries/cloudFormation/rds_public_virtual_interface/test/negative.yaml

```diff
@@ -0,0 +1,35 @@
+#this code is a correct code for which the query should not find any result
+Resources:
+  DBEC2SecurityGroup:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: Open database for access
+      SecurityGroupIngress:
+      - IpProtocol: tcp
+        FromPort: 80
+        ToPort: 80
+        CidrIp: 1.2.3.4/24
+      SecurityGroupEgress:
```

Comment:
`E2521 Property VpcId should exist with SecurityGroupEgress for Resources/DBEC2SecurityGroup/Properties`